site/pages/our-work/case-study/ft/index.js

```diff
@@ -30,7 +30,7 @@ const FinancialTimesCaseStudy = ({ contactUsURL }: CaseStudyFinancialTimesProps)
     <div className={styles.header}>
       <div className={styles.header__container}>
         <div className={styles.header__imageContainer}>
-          <img src={headerImage} alt="shopping cart" className={styles.header__image} />
+          <img src={headerImage} alt="meeting" className={styles.header__image} />
```

Comment:
Also saw this on the link you included

<img width="775" alt="screen shot 2017-08-07 at 16 10 33" src="https://user-images.githubusercontent.com/2177903/29032953-4b366330-7b8b-11e7-9738-3191ffa37b59.png">